mmv1/products/contactcenterinsights/product.yaml

```diff
@@ -1,4 +1,5 @@
 # Copyright 2025 Google Inc.
+
```

Comment:
can you remove this line? Other then that it all looks good!

Reply:
Done